jupyter_console/ptshell.py

```diff
@@ -512,8 +535,8 @@ def interact(self, display_banner=None):
             try:
                 code = self.prompt_for_code()
             except EOFError:
-                if (not self.confirm_exit) \
-                    or ask_yes_no('Do you really want to exit ([y]/n)?','y','n'):
+                if (not self.confirm_exit) or \
```

Comment:
Don't need a \ line continuation inside parens.

Reply:
If i remove the "\" flake 8 tell me that there is a syntax error.
```
jupyter_console\ptshell.py:538:46: E999 SyntaxError: invalid syntax
jupyter_console\ptshell.py:539:25: E113 unexpected indentation
```